<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Todolist</class>
 <widget class="QDialog" name="Todolist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TodoList</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Todolist
{
	background-color:#2D142C;
}
QPushButton
{
	background-color:#801336;
	border-radius:12px;
	border:2px solid #510A32;
}
QPushButton:hover
{
	color: #FFFFFF;
	background-color: #510A32;
}
QPushButton#btn_addtask
{
	border-radius:25px;;
}
QLabel
{
	color:#C72C41;
}
QLineEdit
{
	background-color:#EE4540;
	color:#FFFFFF;
	border-radius:12px;
}
QLineEdit:hover
{
	background-color:#C72C41;
}
QPlainTextEdit
{
	background-color:#510A32;
	color:#C72C41;
	border-radius:12px;
}
QPlainTextEdit:hover#pte_showtask1
{
	background-color:#801336;
}
QPlainTextEdit:hover#pte_showtask2
{
	background-color:#801336;
}
QPlainTextEdit:hover#pte_showtask3
{
	background-color:#801336;
}
QPlainTextEdit:hover#pte_showtask4
{
	background-color:#801336;
}</string>
  </property>
  <widget class="QPushButton" name="btn_addtask">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>670</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>622</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="pte_showtask1">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_showtask2">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_showtask3">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_showtask4">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>771</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_setting">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Setting</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_search">
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>380</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_search">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_sortby">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Sort by</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="rightMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_pervious">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Perv</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_next">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_first">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>First</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_end">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>End</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>121</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_categories">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Categories</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_categories">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>460</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>460</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QLineEdit" name="le_categories">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_categories">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>42</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
